api/preprints/views.py

```diff
@@ -647,3 +649,34 @@ def get_resource(self):
 
     def get_queryset(self):
         return self.get_resource().affiliated_institutions.all()
+
+
+class PreprintInstitutionsRelationship(JSONAPIBaseView, generics.RetrieveUpdateAPIView, PreprintMixin):
```

Comment:
👍 for using the `generics.RetrieveUpdateAPIView` so that we only need to disable `patch()`.